orderer/consensus/hlmirbft/chain.go

```diff
@@ -524,6 +524,26 @@ func (c *Chain) Consensus(req *orderer.ConsensusRequest, sender uint64) error {
 	return nil
 }
 
+//FLy2-64 - proposed changes
+// - check for forward flag in payload
+func checkForwardFlag(reqPayload []byte) bool {
+	forwardBytes := reqPayload[:8]
```

Comment:
Should the / of the flag be included in the check? Go slices select a half-open range with the first index inclusive and last index exclusive so as the flag is of length 9 the / won't currently be included.